internal/runner/rspec_test.go

```diff
@@ -152,3 +153,71 @@ func TestRspecDiscoveryPattern_ExcludePattern(t *testing.T) {
 		t.Errorf("Rspec.discoveryPattern() diff (-got +want):\n%s", diff)
 	}
 }
+
+func TestRspecGetExamples(t *testing.T) {
+	rspec := NewRspec("rspec")
+	files := []string{"fixtures/spec/spells/expelliarmus_spec.rb"}
```

Comment:
This will come from server-side with a leading `./`. Can we confirm this works?

Reply:
Good point! I have updated the test.

Reply:
Ace!